kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/ViewRegistry.kt

```diff
@@ -102,7 +102,12 @@ class ViewRegistry private constructor(
   ): View {
     @Suppress("UNCHECKED_CAST")
     return (allBindings[initialRendering::class] as? ViewBinding<RenderingT>)
-        ?.buildView(this, initialRendering, initialContainerHints, contextForNewView, container)
+        ?.buildView(
+            initialRendering,
+            initialContainerHints + (ViewRegistry to this),
```

Comment:
Isn't this always a no-op? When would `this` be different than the registry already stored in the hints?

Reply:
Good catch, will nuke.